### ISSUE TITLE
REDCap in keyring json

### ISSUE THREAD

**Issue by kcho** [PREDICT-DPACC/lochness#6]
Using the REDCap feature in `lochness` requires two separate fields in the keyring json file as below.

```
{
    "lochness": {
        "REDCAP": {
            "StudyA": {
                "redcap.hcpep": ["hcpep"]
            }
        }
    },

    "redcap.hcpep": {
        "URL": "https://redcap.partners.org/redcap",
        "API_TOKEN": {
            "hcpep": "377F169CB7E2012DE128FAD52C55B6C7"
        }
    },

    ...
}
```

This seems a little redundant.

And the documentation for this part doesn't explain it on a single page (one is in `datasource` and the other is in `keyring` page), which can be quite confusing for readers. Update the documentation.
https://lochness.readthedocs.io/en/latest/data_sources.html#redcap

**Comment by tashrifbillah** [comment]
Hi @kcho , is the `"377F169CB7E2012DE128FAD52C55B6C7"` correct? Will I be able to use that to reproduce the redundancy?

**Comment by kcho** [comment]
@tashrifbillah No, the REDCap API token has already been expired and it was added to the issue text above just to show the `lochness.json` having the redundancy.

You can test with your own API token for the HCPEP project.
